ports/servoshell/egl/servo_glue.rs

```diff
@@ -71,12 +73,16 @@ impl ServoWindowCallbacks {
 }
 
 pub struct ServoGlue {
```

Comment:
I guess this can be renamed to `RunningAppState` to prepare for the combination of the application states of the desktop and embedded versions of servoshell.